### ISSUE TITLE
training time consume

### ISSUE THREAD

**Issue by baiyang4** [OpenPerceptionX/PersFormer_3DLane#51]
Hi there, thanks for your great work! 
I trained with 4 card 3090 on Openlane dataset, set batch to 8. It takes about an hour for a epoch, I'm wondering if the time consuming is normal.

BW

**Comment by RicardLee** [comment]
Hi, it tasks about half an hour on 8 v100 GPUs.  So one hour for one epoch on 4 3090 GPUs is reasonable.

**Comment by baiyang4** [comment]
Thanks for your reply! 
By the way, if I use 4 3090 with batch size 8 per GPU, do I need to train 100 epoch to reach F1=0.47 as stated in this repo?
I've trained 20 epochs with 4 3090, I didn't change learning rate, the F1 is around 43 (20 epoch), and didn't get better after epoch 13. 
I'm wondering if the training config needed to be modified?

BW

**Comment by RicardLee** [comment]
Hi, refer to previous experiment, it will achieve the best score after about 40~50 epochs.

**Comment by baiyang4** [comment]
Thanks for your reply.
I trained 50 epochs on 4 3090, the F-measure is 0.454
I also trained 70 epochs on 8 3090, the F-measure is 0.4447
I didn't change any hyper-params.
I'm wondering if there's any thing wrong with my training.

BW

**Comment by RicardLee** [comment]
Hi, for validating the performance , send your email to us and a best model would share with you.

**Comment by MyEyeAndEar** [comment]
我在3090TI上训练了整整4天100轮，按照代码中默认的参数，但是最终的分数是0.66102999，我也想要最好的模型，可以给我也发一份吗😄. 我的邮箱是z2939562047@163.com

**Comment by RicardLee** [comment]
@MyEyeAndEar i have sent it.

**Comment by MyEyeAndEar** [comment]
Thank you so much！

**Comment by betterCZ101** [comment]
I trained 50 epochs on 4 3090, and the F-measure is 0.4672，Could you please send a best model to me for the further research？
My email is bettercz@163.com
